From 4c3d6b10802b6747ba36b6e7c988f6b584352358 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Wed, 17 Jun 2020 17:42:34 -0400
Subject: [PATCH 1/1] test: disable failing tests.

Just cut them right out of the Makefile if they ain't gonna work.

Bug: https://bugs.gentoo.org/687378
Upstream-bug: https://bugs.netsurf-browser.org/mantis/view.php?id=2779
---
 test/Makefile | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/test/Makefile b/test/Makefile
index 82ffee6..952d238 100644
--- a/test/Makefile
+++ b/test/Makefile
@@ -5,8 +5,6 @@ TESTS := \
 	nsurl \
 	urldbtest \
 	nsoption \
-	bloom \
-	hashtable \
 	hashmap \
 	urlescape \
 	utils \
@@ -47,12 +45,6 @@ messages_SRCS := utils/messages.c utils/hashtable.c test/log.c test/messages.c
 # nsoption test sources
 nsoption_SRCS := utils/nsoption.c test/log.c test/nsoption.c
 
-# Bloom filter test sources
-bloom_SRCS := utils/bloom.c test/bloom.c
-
-# hash table test sources
-hashtable_SRCS := utils/hashtable.c test/log.c test/hashtable.c
-
 # hashmap test sources
 hashmap_SRCS := $(NSURL_SOURCES) utils/hashmap.c utils/corestrings.c test/log.c test/hashmap.c
 hashmap_LD := -lmalloc_fig
-- 
2.26.2

